hack/demo-env/apps/autonomous-guestbook.yaml

```diff
@@ -15,5 +13,7 @@ spec:
     server: https://kubernetes.default.svc
     namespace: guestbook
   syncPolicy:
+    automated:
+      prune: true
```

Comment:
For demo purposes, I think this should stay on manual.

Reply:
Removed as suggested. Thanks.